TestAssets/TestPackages/dotnet-desktop-binding-redirects/dotnet-desktop-binding-redirects.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  
+  <PropertyGroup>
+    <VersionPrefix>1.0.0-rc</VersionPrefix>
+    <TargetFramework>net451</TargetFramework>
+    <AssemblyName>dotnet-desktop-binding-redirects</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <RuntimeIdentifier>win7-x86</RuntimeIdentifier>
```

Comment:
Yes it does. Otherwise we wouldn't be able to support multiple RIDs in the project. Of course, if you use the plural form, then when doing a build you would have to specify which one you want to use.

https://github.com/dotnet/cli/blob/rel/1.0.0/TestAssets/DesktopTestProjects/AppWithProjTool2Fx/App.csproj#L8